config/locales/en/helpers.yml

```diff
@@ -99,6 +101,8 @@ en:
         expert_examination_justification: *ioj_justification_hint
         interest_of_another_justification: *ioj_justification_hint
         other_justification: *ioj_justification_hint
+      steps_income_lost_job_in_custody_form:
+        date_job_lost: For example, 27 3 2007
       steps_evidence_upload_form:
         #TODO update max file size and overall limit
         upload_files: The maximum file size is XMB. Files must be a JPG, BMP, PNG, TIF or PDF.
```

Comment:
Not related to your PR but I think this can be deleted

Reply:
I'll do it in my branch I've started for the next page 👌